app/history.js

```diff
@@ -1,12 +1,27 @@
 import { createHistory, createMemoryHistory } from 'history';
 import { useRouterHistory } from 'react-router';
+import localStorageHistory from './localStorageHistory';
+import { isIOSApp, isBrowser } from './util/browser';
 
 import config from './config';
 
-const isBrowser = typeof window !== 'undefined';
-
-const history = useRouterHistory((isBrowser ? createHistory : createMemoryHistory))({
+const args = {
   basename: config.APP_PATH,
-});
+};
+
+const history = isIOSApp ?
+  // custom local Storage based history for ios app mode
```

Comment:
Should this be moved into app/localStorageHistory.js and exported as `createLocalstorageHistory`?